docs/subvolume.md

```diff
@@ -9,47 +9,47 @@ and delete them without impacting other resources and attached volumes.
 The subvolume command will require the following sub commands:
 * `ls` : [ls](#ls) lists all the subvolumes
     * `--stale`: lists only stale subvolumes
-* `delete <subvolumes> <filesystem> <subvolumegroup>`:
-    [delete](#delete) stale subvolumes as per user's input.
-    It will list and delete only the stale subvolumes to prevent any loss of data.
-    * subvolumes: comma-separated list of subvolumes of same filesystem and subvolumegroup.
-    * filesystem: filesystem name to which the subvolumes belong. 
-    * subvolumegroup: subvolumegroup name to which the subvolumes belong.
+* `delete <filesystem> <subvolume> <subvolumegroup>`:
+    [delete](#delete) stale subvolume as per user's input.
+    It will delete only the stale subvolume to prevent any loss of data.
```

Comment:
How is this preventing loss of data? Can we just keep it simple and remove this line?
```suggestion
```